tests/actions/__snapshots__/render.test.js.snap

```diff
@@ -38,7 +38,7 @@ It may also contain empty lines to break up text into paragraphs.\\";
   <string name=\\"Checkout_OrderOverview_Subtotal_COPY\\">\\"Total price before taxes: %1$s\\"</string>
   <string name=\\"Something_With_An_Arbitrary_Amount_Of_Nesting_COPY\\">\\"We support nesting as deep as you'd like\\"</string>
   <string name=\\"Delivery_Widget_Title_COPY\\">\\"Help: %1$s\\"</string>
-  <string name=\\"Delivery_Widget_SpecialCharacters_COPY\\">\\"The following characters need special handling on some platforms: @ ? &lt; &amp; ' &quot;\\"</string>
+  <string name=\\"Delivery_Widget_SpecialCharacters_COPY\\">\\"The following characters need special handling on some platforms: @ ? &lt; &amp; ' &quot;. And then there is the percent-sign: most cases should be replaced (e.g. 99.9&#37; and &#37;1 and &#37;.1) but not the percentage-sign in strings like %1$s or %2$d.\\"</string>
```

Comment:
@asvitzer: this is what it will do for Android.

Reply:
Thanks, @larslockefeer! Works well :) Sorry for the delay.